matter/efr32/efr32mg24/BRD2601B/autogen/sl_event_handler.h

```diff
@@ -7,5 +7,6 @@ void sl_driver_init(void);
 void sl_service_init(void);
 void sl_stack_init(void);
 void sl_internal_app_init(void);
+void sl_iostream_init_instances(void);
```

Comment:
There is a GSDK issue where this function is present in two different headers. We opted for the latter - can you remove it from here?